scripts/run_backend_tests.py

```diff
@@ -603,7 +603,43 @@ def main(args: Optional[List[str]] = None) -> None:
     if parsed_args.generate_coverage_report:
         subprocess.check_call([sys.executable, '-m', 'coverage', 'combine'])
         report_stdout, coverage = check_coverage(True)
+        print('')
+        print(
+            '+-------------------------------------------------------------'
+            '--------------------------------------------------------------'
+            '-------+'
+            )
+        print(
+            '|----------------------------------------- '
+            'SUMMARY OF THE FILES WITH INCOMPLETE COVERAGE '
+            '------------------------------------------|'
+            )
+        print(
+            '+---------------------------------------------------------------'
+            '-------------------------------------------------------------'
+            '------+'
+            )
+        print('')
         print(report_stdout)
+        if coverage != 100:
+            print(
+                'WARNING: Backend test coverage is below 100%.'
+                ' The rightmost "Missing" column above '
+                'shows which lines are uncovered.'
+                )
```

Comment:
Deindent by 4, similarly in 2 cases below.

Reply:
@seanlip i have deindent these lines and some other lines( print statement below) changes in new updated commit.